src/platform/common/globalActivation.ts

```diff
@@ -53,10 +53,7 @@ export class GlobalActivation implements IExtensionSingleActivationService {
     public async activate(): Promise<void> {
         if (this.dataScienceCodeLensProvider) {
             this.extensionContext.subscriptions.push(
-                vscode.languages.registerCodeLensProvider(
```

Comment:
Running cells doesn't work in IW on the web unless you open a local folder, i.e. if you open a github repo or any other file system it will not work, hence now enabling it for all except notebook cells & IW input cell.